### PR TITLE
Restores ALL missing icon states in /obj/item/clothing/

### PR DESCRIPTION
**Pull request by Ultimate-Fluff**
# About the pull request
This PR restores EVERY. SINGLE. missing icon state in the ```/obj/item/clothing/``` path. No need for reasoning.

# Testings and Procedure
The following list contains all of the articles of clothing with missing icon states:
```
/obj/item/clothing/mask/rebreather/scarf/tacticalmask/squad
/obj/item/clothing/accessory/health
/obj/item/clothing/accessory/health/ceramic_plate
/obj/item/clothing/accessory/health/scrap
/obj/item/clothing/suit/storage/marine/fluff/sas5
/obj/item/clothing/suit/storage/jacket/marine/dress
/obj/item/clothing/head/headband/squad
/obj/item/clothing/head/helmet/marine/fluff/sas
/obj/item/clothing/head/helmet/marine/fluff/coleMercer
/obj/item/clothing/head/helmet/marine/fluff/Bibblesless
/obj/item/clothing/head/helmet/marine/fluff/FernKiller
/obj/item/clothing/head/helmet/skullcap/jungle
/obj/item/clothing/under/color/blue
/obj/item/clothing/under/color/yellow
/obj/item/clothing/under/marine/fluff/juninho
/obj/item/clothing/under/marine/fluff/valentine
/obj/item/clothing/under/serviceoveralls
/obj/item/smartgun_powerpack/merc
```

And here are the list of items restored in order from left to right: Please keep in mind a good chunk of these are donator-exclusive.
image

And they're fully functional. Here are the clothing items worn. Uh, some of these dudes are facing north but they still work.
image

# Changelog

:cl: Puckaboo2
imageadd: Everything under the obj/item/clothing path that had missing icon states now have their icon states again, including all donor clothing.
imageadd: Added all missing Donor clothing icon states.
imageadd: Added a Formal Command uniform. Fancyyy. Don't get this dirty.
code: Fixed some lines of code responsible for flagging some objects as having missing icon states when they already existed.
fix: Added icon states that have been deleted in previous PRs of long ago.
fix: Jungle Cowls will now have their proper camouflage in jungle maps instead of using snow for all maps.
fix: Metal, ceramic, and scrap plates are functional and can now be slipped into armor.
fix: Yellow and Blue jumpsuits exist and can now be worn.
fix: The M44R Holster and the Medical Storage Rig will now properly appear in the map editor.
del: Deleted some unnecessary and outdated objects that were flagged as having missing icon states.
del: Removed a few duplicate icon states. 
/:cl:

### CODE REVIEW

**Review by Agameofscones — APPROVED** [review]
this makes the linter happy. Sprites approved

File: code/modules/cm_tech/implements/armor.dm [hunk 1]
```diff
@@ -2,10 +2,10 @@
 /obj/item/clothing/accessory/health
 	name = "armor plate"
 	desc = "A metal trauma plate, able to absorb some blows."
-
 	icon = 'icons/obj/items/items.dmi'
+	icon_state = "regular2_100"
 	var/base_icon_state
-	icon_state = "regular2"
+	base_icon_state = "regular2"
```
Comment (harryob) [inline code/modules/cm_tech/implements/armor.dm:8]:
```suggestion
```

File: code/modules/cm_tech/implements/armor.dm [hunk 2]
```diff
@@ -2,10 +2,10 @@
 /obj/item/clothing/accessory/health
 	name = "armor plate"
 	desc = "A metal trauma plate, able to absorb some blows."
-
 	icon = 'icons/obj/items/items.dmi'
+	icon_state = "regular2_100"
 	var/base_icon_state
```
Comment (harryob) [inline code/modules/cm_tech/implements/armor.dm:7]:
```suggestion
	var/base_icon_state = "regular2"
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Comment by Agameofscones** [comment]
Of note, ceramic plates aren't supposed to be wearable. They're only supposed to be able to slot onto armor as an accessory

**Pull request merged**